src/data_importing/httpImpl.ts

```diff
@@ -196,11 +196,11 @@ export class HttpImpl implements DataSource {
             const lastImport = await ImportStorage.Instance.RetrieveLast(this.dataSourceT.id!);
 
             let lastImportTime = "";
-            if(!lastImport.isError && lastImport.value.stopped_at != null) {
-                lastImportTime = lastImport.value.stopped_at.toUTCString()
+            if(!lastImport.isError && lastImport.value.status === "completed") {
```

Comment:
Is this project making use of enums? If so, would this be a good candidate? If not, has that design decision been made explicit in documentation?

Reply:
unfortunately I don't think I can use an enum type with `io-ts`, that's why I've stuck with string literals.